examples/Remarketing/UploadStoreSalesTransactions.cs

```diff
@@ -0,0 +1,488 @@
+            UserData userDataWithEmailAddress = new UserData()
+            {
+                UserIdentifiers =
+                {
+                    new UserIdentifier()
+                    {
+                        // Email addresses must be normalized and hashed.
+                        HashedEmail = NormalizeAndHash("customer@example.com")
+                    },
+                    new UserIdentifier()
+                    {
+                        AddressInfo = new OfflineUserAddressInfo()
+                        {
+                            State = "NY"
+                        }
+                    },
+                },
+                TransactionAttribute = new TransactionAttribute()
+                {
+                    ConversionAction =
+                        ResourceNames.ConversionAction(customerId, conversionActionId),
+                    CurrencyCode = "USD",
+                    // Converts the transaction amount from $200 USD to micros.
+                    TransactionAmountMicros = 200L * 1_000_000L,
+                    // Specifies the date and time of the transaction. This date and time will be
+                    // interpreted by the API using the Google Ads customer's time zone.
+                    // The date/time must be in the format "yyyy-MM-dd hh:mm:ss".
+                    TransactionDateTime = "2020-05-01 23:52:12"
+                }
+            };
+
+            // Creates the second transaction for upload based on a physical address.
+            UserData userDataWithPhysicalAddress = new UserData()
+            {
+                UserIdentifiers =
+                {
+                    new UserIdentifier()
+                    {
+                        AddressInfo = new OfflineUserAddressInfo()
+                        {
+                            // Names must be normalized and hashed.
+                            HashedFirstName = NormalizeAndHash("John"),
+                            HashedLastName = NormalizeAndHash("Doe"),
+                            CountryCode = "US",
+                            PostalCode = "10011"
+                        }
+                    }
+                },
+                TransactionAttribute = new TransactionAttribute()
+                {
+                    ConversionAction =
+                        ResourceNames.ConversionAction(customerId, conversionActionId),
+                    CurrencyCode = "EUR",
+                    // Converts the transaction amount from 450 EUR to micros.
+                    TransactionAmountMicros = 450L * 1_000_000L,
+                    // Specifies the date and time of the transaction. This date and time will be
+                    // interpreted by the API using the Google Ads customer's time zone.
+                    // The date/time must be in the format "yyyy-MM-dd hh:mm:ss".
+                    TransactionDateTime = "2020-05-14 19:07:02"
```

Comment:
Is this date somehow special? If not, can we do something more generic, like

DateTime.Today.AddDays(-1).ToString("yyyy-MM-dd HH:mm:ss"); Same comment for line 360 above.

Reply:
I don't think they're special; I just took them from the golden.  Updated both.